SnapsInAZfs/ConfigConsole/SnapsInAZfsConfigConsole.cs

```diff
@@ -38,13 +38,12 @@ public SnapsInAZfsConfigConsole( )
         Ready += SnapsInAZfsConfigConsoleOnReady;
         InitializeComponent( );
         globalConfigMenuItem.Action = ShowGlobalConfigurationWindow;
-        globalConfigMenuItem.Shortcut = Key.CtrlMask | Key.g;
+        globalConfigMenuItem.Shortcut = Key.CtrlMask | Key.G;
```

Comment:
It's not necessary because only the Key.CtrlMask | Key.G will be handled. Try put them separated with breakpoints and you'll see. Because the Ctrl key is catch as 1 to 26 and to get the corresponding other key we add 64. So 1+64=65 which is the 'A' char and 26+64=90 which is the 'Z' that are all as upper case.
I am regret to have created the keys of lowercase letters, because it induces users to generate bad keys. Maybe I propose remove them from V2. See https://github.com/gui-cs/Terminal.Gui/issues/2826

Reply:
This is helpful to know and definitely worth documenting in the API documentation, if it isn't already (I don't think it is, for V1)

Reply:
> This is helpful to know and definitely worth documenting in the API documentation, if it isn't already (I don't think it is, for V1)

Well this a functional API and for the end user it's needed how it work, but documenting that they should avoid lower cases, yes. Unfortunately I'm not so good to deal with documentation. If you want to provide some help with that on the Terminal.Gui I really appreciate 😃